### PR TITLE
Improve Svelte LP

### PR DESCRIPTION
**Pull request by marcoow**
* change wording slightly for SEO
* track clicks for 1:1 call bookings
* add section on custom development
* change "Training" to "Training and Support"

### CODE REVIEW

File: src/svelte-consulting.njk [hunk 1]
```diff
@@ -48,21 +48,31 @@ permalink: /svelte-consulting/
...
+  "linkClass": "plausible-event-name=Svelte+1on1+Call",
   "linkText": "Book a free 1:1 call with Paolo"
 } %}
 
 {% set 'slide_2' = {
   "color": "black",
-  "title": "Training",
-  "text": "Svelte has excellent documentation for diving straight into the framework. However, sometimes active learning in a workshop taught by a subject matter expert can accelerate the learning experience. Our trainers are world-class mentors and can design a learning path that best fits your needs.",
+  "title": "Custom Development",
+  "text": "We design and develop complete web apps in Svelte Rust so your team can focus on other initiatives. We are experts in Svelte and SvelteKit as well as related technology like Tailwind, Storybook, and testing with Puppetter etc. Our team delivers high quality codebases as well as the tooling and infrastructure needed for sustainable maintenance.",
```
Comment (SaeLLo) [inline src/svelte-consulting.njk]: Delete "Rust"?

File: src/svelte-consulting.njk [hunk 2]
```diff
@@ -48,21 +48,31 @@ permalink: /svelte-consulting/
...
+  "text": "Svelte has excellent documentation for diving straight into the framework. However, sometimes active learning in a workshop taught by a subject matter expert can accelerate the learning experience or help overcome challenges. Our trainers are world-class mentors and can design a learning path that best fits your needs.",
   "linkUrl": "/services/workshops/svelte-and-sveltekit/",
   "linkClass": "plausible-event-name=Svelte+Workshop",
   "linkText": "Check out our Svelte & SvelteKit workshop!"
 } %}
 
-{% set 'slide_3' = {
+{% set 'slide_4' = {
   "color": "aqua",
   "title": "Team augmentation",
```
Comment (SaeLLo) [inline src/svelte-consulting.njk]: Titels are not consistent. It should be Augmentation

File: src/svelte-consulting.njk [hunk 3]
```diff
@@ -36,7 +36,7 @@ permalink: /svelte-consulting/
     <div class="split-content__content">
       <div class="split-content__text">
         <p>We’re frontend experts – our team speaks at international events, is <a href="/this-week-in-open-source/">actively contributing to a number of open source projects</a> and our Svelte expert Paolo Ricciuti is a Svelte ambassador and the co-author of <a href="https://www.sveltelab.dev/">SvelteLab</a>.</p>
```
Comment (SaeLLo) [inline src/svelte-consulting.njk:38]: Titel should be "A Unique Skillset"

Comment (marcoow) [reply src/svelte-consulting.njk:38]: fixed

File: src/svelte-consulting.njk (hunk 2 above)
Comment (marcoow) [reply src/svelte-consulting.njk:77]: fixed

File: src/svelte-consulting.njk (hunk 1 above)
Comment (marcoow) [reply src/svelte-consulting.njk]: 🤦

**Review by SaeLLo — APPROVED** [review]

**Review by SaeLLo — APPROVED** [review]

File: src/svelte-consulting.njk [hunk 4]
```diff
@@ -48,31 +48,41 @@ permalink: /svelte-consulting/
...
+  "linkClass": "plausible-event-name=Svelte+1on1+Call",
   "linkText": "Book a free 1:1 call with Paolo"
 } %}
 
 {% set 'slide_2' = {
   "color": "black",
-  "title": "Training",
-  "text": "Svelte has excellent documentation for diving straight into the framework. However, sometimes active learning in a workshop taught by a subject matter expert can accelerate the learning experience. Our trainers are world-class mentors and can design a learning path that best fits your needs.",
+  "title": "Custom Development",
+  "text": "We design and develop complete web apps in Svelte so your team can focus on other initiatives. We are experts in Svelte and SvelteKit as well as related technology like Tailwind, Storybook, and testing with Puppetter etc. Our team delivers high quality codebases as well as the tooling and infrastructure needed for sustainable maintenance.",
```
Comment (paoloricciuti) [inline src/svelte-consulting.njk]: I'd say playwright rather than puppeteer

Comment (paoloricciuti) [reply src/svelte-consulting.njk]:
```suggestion
  "text": "We design and develop complete web apps in Svelte so your team can focus on other initiatives. We are experts in Svelte and SvelteKit as well as related technology like Tailwind, Storybook, and testing with Playwright etc. Our team delivers high quality codebases as well as the tooling and infrastructure needed for sustainable maintenance.",
```

Comment (marcoow) [reply src/svelte-consulting.njk]: ugh, sure 🤦

**Review by SaeLLo — APPROVED** [review]

**Pull request merged**